entproto/cmd/protoc-gen-entgrpc/converter.go

```diff
@@ -90,29 +92,67 @@ func (g *serviceGenerator) newConverter(fld *entproto.FieldMappingDescriptor) (*
 	}
 	return out, nil
 }
+func convertPbMessageType(fqn, entFieldType string, conv *converter) error {
+	switch {
+	case fqn == "google.protobuf.Timestamp":
+		conv.toProtoConstructor = protogen.GoImportPath("google.golang.org/protobuf/types/known/timestamppb").Ident("New")
+	case isWrapperType(fqn):
+		typ := strings.Split(fqn, ".")[2]
+		constructor := strings.TrimSuffix(typ, "Value")
+		conv.toProtoConstructor = protogen.GoImportPath("google.golang.org/protobuf/types/known/wrapperspb").Ident(constructor)
+		goType := wrapperPrimitives[fqn]
+		if entFieldType != goType {
+			conv.toProtoConversion = goType
+		}
+		conv.toEntModifier = ".GetValue()"
+	default:
+		return fmt.Errorf("entproto: no mapping for pb field type %q", fqn)
+	}
+	return nil
+}
 
-func (g *serviceGenerator) renderToProto(fc *converter, ident string) string {
+func (g *serviceGenerator) renderToProto(conv *converter, ident string) string {
```

Comment:
Why?

Reply:
fc used to be "fieldCodec" and i missed changing the parameter name